travis/build-all-in-one-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Only build docker container for master branch
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo 'upload to Docker Hub'; else echo 'skip docker upload for PR'; exit 0; fi
```

Comment:
shouldn't this also check for master branch?  if another branch is pushed, I assume it'll also be built without PR